src/helpers.js

```diff
@@ -154,9 +153,25 @@ export function normalizeSwagger(parsedSpec) {
               if (!operation[inhName]) {
                 operation[inhName] = inherits[inhName]
               }
+              else if (inhName === "parameters") {
+                var exists = false
+                for (const item in inherits[inhName]) {
+
+                  for (const param in operation[inhName]) {
+                    if (operation[inhName][param].name === inherits[inhName][item].name) {
+                      exists = true
+                    }
+                  }
+
+                  if (!exists) {
```

Comment:
This looks wrong because `exists` is never reset. So after the first `exists = true`, all subsequent params will be be pushed to operation.